net/frr/Makefile

```diff
@@ -8,15 +8,15 @@
 include $(TOPDIR)/rules.mk
 PKG_NAME:=frr
 PKG_VERSION:=8.1.0
-PKG_RELEASE:=1
-PKG_SOURCE_DATE:=2021-11-12
+PKG_RELEASE:=2
```

Comment:
hmm?

Reply:
it's still 8.1.0 branch, no official 8.1.1 release